### DISCUSSION TITLE
https://developer.bentley.com/my-apps/ to access Bentley ALIM API

### DISCUSSION THREAD

**Discussion by PeterJBassett** [iTwin/itwinjs-core#2247] [Q&A]
Hello - I have a question. Is it possible to configure an SPA app created on https://developer.bentley.com/my-apps/ to access the Bentley ALIM API (See https://datastorage.dev.aw-au.bentley.com/documentation/index.html#documents) ?

**Comment by aruniverse** [comment]
Take a look at the [ALIM authentication](https://datastorage.dev.aw-au.bentley.com/documentation/index.html?shell#authentication) section. As long as you can get a valid access token for their service, you should be able to connect your iTwin App to their service. The ALIM api's are not apart of iTwin Platform, so I'm assuming you'll need to have a different Auth Client to make requests to that service, with its own Issuing Authority (probably `imsoidc.bentley.com`) along with whatever specific scopes for that service. You're best option is to reach out directly to someone who works on the ALIM apis.

**Reply by PeterJBassett** [reply]
Thank you for your reply, do you know of a contact within Bentley who works on the ALIM APIs?

**Comment by PeterJBassett** [comment]
Hello Arun, thank you for sending a message via Teams, unfortunately, I can't read the message in Teams, I just got an email about the message.
We have been working with some people in Bentley on iModeljs for synchronisations, but don’t really know who to contact in Bentley for help on the ALIM API - specifically on how to authenticate and obtain an access token for the ALIM API.
